mavlink/modules/flight_input_device.py

```diff
@@ -0,0 +1,34 @@
+from dronekit import connect 
+
+class InputDevice: 
```

Comment:
Rename to `FlightInputDevice` as `InputDevice` is too generic. It is always a good habit to be specific with your class and variable names so other people get a better idea on what the class/variable is for.